lib/pages/wp-home-page.js

```diff
@@ -35,9 +36,9 @@ export default class WPHomePage extends BaseContainer {
 		return d.promise;
 	}
 	checkURL( culture ) {
-		const target = culture
-			? culture + '.wordpress.com'
-			: 'wordpress.com' ;
+		const target = culture ?
```

Comment:
I noticed a couple linting warnings for this line and the line below it — the characters `?` and `:` should be a the beginning of the line.